App/Config/ReduxPersist.ts

```diff
@@ -90,13 +90,26 @@ const migrations: MigrationManifest = {
         }
       }
     }
+  },
+  7: (persistedState) => {
+    const state = persistedState as any
+    return {
+      ...state,
+      preferences: {
+        ...state.preferences,
+        tourScreens: {
+          ...state.preferences.tourScreens,
+          threads: true
```

Comment:
Converted the threads onboarding screen into the same as the wallet one. now that threads are the default view, it was creating a flash of onboarding every time it loaded with the old way anyway